circleci/images/pgupgradetester/files/sbin/install-testdeps

```diff
@@ -47,16 +47,11 @@ echo "deb http://apt.postgresql.org/pub/repos/apt/ ${codename}-pgdg main ${pg_la
 pkg_cloud_url="https://packagecloud.io/install/repositories/citus-bot/the-process/script.deb.sh"
 curl -sf "${pkg_cloud_url}" | unique_id=exttester bash
 
-cat > /etc/apt/preferences.d/citusdata_the-process.pref <<EOF
```

Comment:
We don't need to use our package here because postgres upgrade tests don't use isolation tester or anything special.